deploy/crds/toolchain_v1alpha1_usersignup_crd.yaml

```diff
@@ -45,8 +45,8 @@ spec:
       name: ApprovedBy
       priority: 1
       type: string
-    - jsonPath: .spec.deactivated
-      name: Deactivated
+    - jsonPath: .spec.states
```

Comment:
Please rebase.

Reply:
yes, I'm not sure what happened here. My apologies, I'll fix/clean that

Reply:
it's fixed now 😅